### DISCUSSION TITLE
About public sponsor Badge

### DISCUSSION THREAD

**Discussion by ipetersenpai** [community/community#57941] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

If I stop sponsoring in github open source/org, my public sponsor badge will also get remove on my profile?

**Comment by melvincwng** [comment, marked as answer]
@ipetersenpai, based on my understanding, the public sponsor badge/achievement should still remain in your GitHub profile even if you stop sponsoring that particular individual or organization.

Hope this clarifies and answers your question 😃

**Comment by BayaInnovation** [comment]
how many days is github sponsor badge?

**Reply by ipetersenpai** [reply]
Base on my experience after the 30 days of my active sponsorship, the badge still remain on my profile.

**Reply by Tyo-Dev** [reply]
yes bro i agree with your opinion

**Comment by RaflyLesmana3003** [comment]
The GitHub sponsor badge is typically active for 30 days. It is a way for sponsors to showcase their support for a project or developer on GitHub. During this period, sponsors' badges are displayed prominently on the sponsored user's profile and repositories to highlight their contribution. After 30 days, the badge will no longer be displayed unless the sponsorship is renewed or maintained.

**Reply by Tyo-Dev** [reply]
I thought the badge would be displayed permanently after we got it.
